### ISSUE TITLE
Implement sidecar update

### ISSUE THREAD

**Issue by Stebalien** [ipfs/testground#171]
We should have a `testground sidecar update --runner=...` to deploy a new version of the sidecar.

**Comment by daviddias** [comment]
Shouldn't this happen through the ansible scripts that configured the instantiated infra created by Terraform??

**Comment by raulk** [comment]
@daviddias not really, because this service is managed by Docker Swarm, as a system service. So we'd use the `docker service update` command as explained here: https://docs.docker.com/engine/swarm/swarm-tutorial/rolling-update/

**Comment by daviddias** [comment]
Gotcha, so to understand it fully, what you are saying is that it is better to:

- create a ssh tunnel and call a `docker service update` locally

instead of: 

- calling an Ansible script that ssh into the machine and calls `docker service update` there
